tests/tests/test_basic_integration.py

```diff
@@ -139,12 +139,50 @@ def deployment_callback():
                 template = run(catcmd)
                 wait_count = 0
 
+    @pytest.mark.timeout(1000)
+    @pytest.mark.usefixtures("standard_setup_one_client_bootstrapped")
+    def test_forced_inventory_update_from_client(self):
+        """Forces an inventory update from an idling client."""
+        if not env.host_string:
+            execute(self.test_forced_inventory_update_from_clien,
+                    hosts=get_mender_clients())
+            return
+
+        # Give the image a really large wait interval.
+        sedcmd = "sed -i.bak 's/%s/%s/' /etc/mender/mender.conf" % ("\(.*PollInter.*:\)\( *[0-9]*\)", "\\1 1800")
+        out = run(sedcmd)
+        if out.return_code != 0:
+            logger.error(out)
+            pytest.fail("failed to set a large polling interval for the client.")
```

Comment:
If `run()` receives a non-zero return code, it throws an exception, so this code will never be hit.

Reply:
http://docs.pyinvoke.org/en/1.2/api/runners.html#invoke.runners.Runner.run
http://docs.pyinvoke.org/en/1.2/api/runners.html#invoke.runners.Result
?